src/app.ts

```diff
@@ -65,7 +65,9 @@ export default class App {
 
     app.use(express.static(path.join(__dirname)));
 
-    app.get('/', (_req, res) => res.render('pages/index.ejs'));
+    app.get('/', (_req, res) => {
+      res.render('pages/index.ejs', { key: ConfigService.getStringValue('inspectlet.wid') ? ConfigService.getStringValue('inspectlet.wid') : '' });
```

Comment:
`key` is quite global name given but it should be scoped to inspectlet only. Consider renaming  it to`inspectlet_key`